index.html

```diff
@@ -3158,7 +3158,10 @@ <h2 id="AudioDestinationNode">
           Giving a concrete example, if the audio hardware supports 8-channel
           output, then we may set <a href=
           "#widl-AudioNode-channelCount">channelCount</a> to 8, and render 8
-          channels of output.
+          channels of output. <span class="synchronous">An
+          <code>NotSupportedError</code> MUST be fired if the implementation
+          does not support changing the <a href=
+          "#widl-AudioNode-channelCount">channelCount</a></span>.
```

Comment:
The previous sentence (not shown here) already says we thrown an error for invalid values of `channelCount`.  Are you saying that I can have, say a 7 channel device and I'm not allowed to set `channelCount` to any value from 1 to 7?